lib/pages/settings_screen.dart

```diff
@@ -34,7 +37,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
       appBar: AppBar(
         backgroundColor: const Color(0xFF1A1A1A),
         title: const Text(
-          'Settings',
+          'Profile',
```

Comment:
Perhaps we could change the App Bar title to Settings and create a heading named Profile below the app bar